### PR TITLE
LKE-353: Add basic k8s documentation

### PR DESCRIPTION
**Pull request by enys**

### CODE REVIEW

File: content/admin/content/getting-started/start.md [hunk 1]
```diff
@@ -77,3 +77,10 @@ Here is an example:
      --user linkurious:1000 \
      linkurious:{{package.version}}
 ```
+## Kubernetes
+
+Please read the previous section on starting a {{lke}} instance [using docker](#docker-linux), and the section on [fault tolerance](/faq/#how-can-fault-tolerance-be-achieved)
+
+A simple way to test out {{lke}} using kubernetes is to create a simple deployement, using only one replica, and alocate a [PersitentVolume](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/) for both of the volumes (lke-data, lke-elasticsearch) described above.
```
Comment (francescoinfante) [inline content/admin/content/getting-started/start.md:84]: 2 typos:  in alocate and Persitent.

File: content/admin/content/getting-started/start.md [hunk 2]
```diff
@@ -77,3 +77,10 @@ Here is an example:
...
+
+In production however you would want to follow the [fault tolerance guide](/faq/#how-can-fault-tolerance-be-achieved) and use a satefull set, with a  main/failover strategy, and the appropriate strategy configured for your load-balancer or [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
```
Comment (francescoinfante) [inline content/admin/content/getting-started/start.md:86]: "In production, however,"

what satefull means? (external link maybe)

Comment (enys) [reply content/admin/content/getting-started/start.md:86]: ah yep, it's a 'type' of deployement in kubernetes vs classical deployements. Will add the link

Comment (pastahito) [inline content/admin/content/getting-started/start.md:86]: `with a  main/failover` <- there's a double space here

File: content/admin/content/getting-started/start.md [hunk 3]
```diff
@@ -77,3 +77,10 @@ Here is an example:
      --user linkurious:1000 \
      linkurious:{{package.version}}
 ```
+## Kubernetes
+
+Please read the previous section on starting a {{lke}} instance [using docker](#docker-linux), and the section on [fault tolerance](/faq/#how-can-fault-tolerance-be-achieved)
```
Comment (pastahito) [inline content/admin/content/getting-started/start.md:82]: Missing final period `.`

File: content/admin/content/getting-started/start.md (hunk 2 above)
Comment (enys) [reply content/admin/content/getting-started/start.md:86]: we need a linter :D

**Comment by enys** [comment]
@pastahito @francescoinfante Does the content make sense, especially on the failover part ?

**Pull request merged**